crypto/store/loader_file.c

```diff
@@ -764,20 +764,22 @@ static OSSL_STORE_LOADER_CTX *file_open(const OSSL_STORE_LOADER *loader,
         }
     } else {
         BIO *buff = NULL;
-        char peekbuf[4096];
+        char *peekbuf = OPENSSL_malloc(4096);
```

Comment:
Ok then...  but with a grudge, I really hate to have to use the heap for this...  bu ah well